packages/@react-aria/overlays/test/calculatePosition.test.ts

```diff
@@ -138,13 +138,16 @@ describe('calculatePosition', function () {
     };
 
     const container = createElementWithDimensions('div', containerDimensions);
+    Object.assign(container.style, {
```

Comment:
calculate position will look for the containing block, which should be `container` and not document.body as it is in the default case. The default case should mostly be covered by chromatic. These tests cover our other main case where the overlay is inside the container that calculate position will use.